examples/gallery/demos/bokeh/image_range_tool.ipynb

```diff
@@ -0,0 +1,109 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from itertools import islice\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import holoviews as hv\n",
+    "from holoviews.plotting.links import RangeToolLink\n",
+    "\n",
+    "hv.extension('bokeh')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This demo demonstrates how to link two plots using the RangeToolLink along both axes. This can be useful to get an overview and a detailed view of some data at the same time."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Declare data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def mandelbrot_generator(h,w, maxit, bounds):\n",
+    "    \"Generator that yields the mandlebrot set.\"\n",
+    "    (l,b,r,t) = bounds\n",
+    "    y,x = np.ogrid[b:t : h*1j, l:r:w*1j]\n",
+    "    c = x+y*1j\n",
+    "    z = c\n",
+    "    divtime = maxit + np.zeros(z.shape, dtype=int)\n",
+    "    for i in range(maxit):\n",
+    "        z  = z**2 + c\n",
+    "        diverge = z*np.conj(z) > 2**2\n",
+    "        div_now = diverge & (divtime==maxit)\n",
+    "        divtime[div_now] = i\n",
+    "        z[diverge] = 2\n",
+    "        yield divtime\n",
+    "\n",
+    "def mandelbrot(h,w, n, maxit, bounds):\n",
+    "    \"Returns the mandelbrot set computed to maxit\"\n",
+    "    iterable =  mandelbrot_generator(h,w, maxit, bounds)\n",
+    "    return next(islice(iterable, n, None))\n",
+    "\n",
+    "bounds = (-2,-1.4,0.8,1.4)\n",
+    "mbset = mandelbrot(800,800, 45, 46, bounds)\n",
+    "\n",
+    "mbset_image = hv.Image(mbset, bounds=bounds)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Declare plot"
```

Comment:
It is a neat example but it needs a sentence here to explain what is going on and what the user should try to see the linked ranges in action...

Reply:
Agreed, at some point someone will have to go through all the demos and add some explanation. But you're right that new examples should have some text.

Reply:
Same issue with ``timeseries_range_tool.ipynb``...